test/integration/scheduler/queue_test.go

```diff
@@ -336,8 +336,21 @@ func TestCustomResourceEnqueue(t *testing.T) {
 	}
 
 	// Now we should be able to pop the Pod from activeQ again.
-	podInfo = testCtx.Scheduler.NextPod()
-	if podInfo.Attempts != 2 {
-		t.Errorf("Expected the Pod to be attempted 2 times, but got %v", podInfo.Attempts)
+	done := make(chan struct{})
```

Comment:
how is this related? Perhaps it's worth its own PR?

Reply:
> Perhaps it's worth its own PR?

SG. I will make it a separate one.

> how is this related?

Not quit. It's added to speed up surfacing the failure case of this integration test. For example, if a PR introduces a bug, the integration test won't pass for sure, but it hanged there (due to `Pop()` 'd hang), and time out when the global integration test's timeout (10m) is reached.